AdyenDropIn/Presentation/WrapperViewController.swift

```diff
@@ -61,7 +61,7 @@ internal final class WrapperViewController: UIViewController {
 
 extension ModalViewController {
 
-    private var leastPresentableHeightScale: CGFloat { 0.3 }
+    private var leastPresentableHeightScale: CGFloat { 0.25 }
```

Comment:
maybe we don't really need a bottom limit 🤔

Reply:
maybe 🤔 i had to decrease it to match the design.